lib/graph.py

```diff
@@ -166,7 +173,9 @@ def _make_cmd_node(complete, outcome, command):
             raise RuntimeError("Unknown outcome '%s'" % outcome)
         else:
             cmd_style["fillcolor"] = "#eceff1"
-        return lib.graph.CommandNode(command, **cmd_style)
+        return lib.graph.CommandNode(
+            pipeline_name, description, command, **cmd_style
```

Comment:
Could we retain the current style here? The `)` doesn't need to be on a separate line. Same below.